src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -357,6 +357,51 @@ await foreach (var streamValue in asyncEnumerable)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
+        [LogLevel(LogLevel.Trace)]
```

Comment:
Why do we need trace logs for this test but not the whole FunctionalTests project?

Reply:
This particular test doesn't need trace logs. We just following the surrounding tests.

Reply:
@JunTaoLuo Do you know how to change the default min log level for the entire test project?

Reply:
We only really need Trace when the test is flaky and we're gathering more data. By default the level is Debug, I think it's fine to remove this and keep the default.